test/src/templates/test_utils.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter_deck/flutter_deck.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+class SlideTester {
+
```

Comment:
This is not needed.

Reply:
the package flutter test is not needed or having it as a class not needed?

Reply:
Sorry, I was not clear here. The empty line between class SlideTester and it's constructor is not needed.